### PR TITLE
Chassis PR

### PR DESCRIPTION
**Pull request by Jason-VL**

### CODE REVIEW

**Review by Rylan-Meilutis — APPROVED** [review]
Looks good

File: src/main/java/frc/robot/subsystems/Chassis.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+public class Chassis {
+    private static final int TIMEOUT = 0;
+    public static final double WHEEL_RADIUS_INCHES_MECANUM = 3;
+    public static final double MAX_SPEED_FPS_TRACTION = 9.67 * 1.01;
+    public static final double MAX_TICKS_PER_100MS = MAX_SPEED_FPS_TRACTION * 4096.0 / (Math.PI * WHEEL_RADIUS_INCHES_MECANUM * 2.0 / 12.0) / 10.0;
+
+    private static final int FL_motor = 1;
+    private static final int FR_motor = 2;
+    private static final int RL_motor = 3;
+    private static final int RR_motor = 4;
```
Comment (mmarchetti) [inline src/main/java/frc/robot/subsystems/Chassis.java]: Do these constants duplicate the ones in wiring.Java?

File: src/main/java/frc/robot/subsystems/Chassis.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        kP = 6e-5;
+        kI = 0;
+        kD = 0;
+        kIz = 0;
+        kFF = 0.000015;
+        kMaxOutput = 1;
+        kMinOutput = -1;
+        maxRPM = 5700;
+
+        FL_pidController.setP(kP);
```
Comment (mmarchetti) [inline src/main/java/frc/robot/subsystems/Chassis.java]: A helper method to create a CANSparkMax, reset defaults, and configure the PID controller would eliminate a lot of duplication here.

**Review by mmarchetti — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/Chassis.java (hunk 1 above)
Comment (Rylan-Meilutis) [reply src/main/java/frc/robot/subsystems/Chassis.java]: They still need to be moved into the wiring class

File: src/main/java/frc/robot/subsystems/Chassis.java (hunk 2 above)
Comment (Rylan-Meilutis) [reply src/main/java/frc/robot/subsystems/Chassis.java]: Agreed, I will take care of that

Comment (mmarchetti) [reply src/main/java/frc/robot/subsystems/Chassis.java]: Looks good!

**Pull request merged**